docs/qa/v034/README.md

```diff
@@ -0,0 +1,251 @@
+
+These experiments and all others below the diagonal they form have in common that the total
+number of transactions processed is noticeably less than the product $c \cdot r \cdot 90$,
+which is the expected number of transactions when the system is able to deal well with the
+load.
+
+At this point, we chose an experiment with an important load but below the saturation point,
+in order to further study the performance of this release.
+**The chosen experiment is `r=400,c=8`**.
+
+This is a plot of the CPU load of the load runner for `r=400,c=8`,
+where we can see that the load (average over 1 minute) stays below 1 most of the time.
+
+![load-load-runner](./img/v034_r400c8_load-runner.png)
+
+## Examining latencies
+
+The method described [here](../method.md) allows us to plot the latencies of transactions
+for all experiments.
+
+![all-latencies](./img/v034_r400c8_latencies.png)
+
+As we can see, even the experiments beyond the saturation diagonal managed to keep
+transaction latency stable (i.e. not constantly increasing).
+Our interpretation for this is that contention within Tendermint was propagated,
+via the websockets, to the load runner,
+hence the load runner could not produce the target load, but a fraction of it.
+
+Further examination of the Prometheus data, showed that the mempool contained many transactions
+at steady state, but did not grow much without quickly returning to this steady state. This demonstrates
+that the transactions were able to be processed by the Tendermint network at least as quickly as they
+were submitted to the mempool. Finally, the test script made sure that, at the end of an experiment, the
+mempool was empty so that all transactions submitted to the chain were processed.
+
+This plot can we used as a baseline to compare with other releases.
+
+## Prometheus Metrics on the Chosen Experiment
+
+As mentioned [above](#finding-the-saturation-point), the chosen experiment is `r=400,c=8`.
+This section further examines key metrics for this experiment extracted from Prometheus data.
+
+### Mempool Size
+
+The mempool size, a count of the number of transactions in the mempool, was shown to be stable and homogeneous
+at all full nodes. It did not exhibit any unconstrained growth.
+The plot below shows the evolution over time of the cumulative number of transactions inside all full nodes' mempools.
+The two spikes that can be observed correspond to a period where consensus instances reached round 1
+at some nodes.
+
+![mempool-cumulative](./img/v034_r400c8_mempool_size.png)
+
+The plot below shows evolution of the average over all full nodes, which oscillate around 140 outstanding transactions.
+
+![mempool-avg](./img/v034_r400c8_mempool_size_avg.png)
+
+The peaks observed coincide with the moments when some nodes process to round 1 of consensus (see below).
+
+### Peers
+
+The number of peers was stable at all nodes.
+It was higher for the seed nodes (around 140) than for the rest (between 21 and 71).
```

Comment:
In thesis, a regular (non-seed) node should have between 10 and 50 peers. How can they reach more than 70 peers?

Reply:
I don't know. We're using default config in these testnets.
We're likely going to re-do these tests. Do you want to be in the loop?

Reply:
Yes, I'm in. Where is the adopted config file?

The `DefaultP2PConfig` produces:
                MaxNumInboundPeers:           40,
                MaxNumOutboundPeers:          10,
So, 50 should be the upper bound in the number of peers. However, there is a [fix](https://github.com/tendermint/tendermint/pull/3762) introduced in the p2p layer to dial all addresses retrieved from a seed node that ignores these limits (the outbound peers limit, in the case). I commented this [here](https://github.com/tendermint/tendermint/blob/spec/p2p-v0.34/spec/p2p/v0.34/peer_manager.md#fast-dialing), and maybe it deserves an issue explaining the scenario.

Reply:
OK. Will keep this comment open until we extract the new results. Then I will add a paragraph explaining this

Reply:
Created #9548 point out this observation.